yrs/src/store.rs

```diff
@@ -352,6 +352,10 @@ impl StoreRef {
     pub fn try_borrow_mut(&self) -> Result<RefMut<Store>, BorrowMutError> {
         self.0.try_borrow_mut()
     }
+
+    pub fn weak_ref(&self) -> Weak<RefCell<Store>> {
```

Comment:
This is related to #208 . Hence so far both document and branches have hold strong references to store, in result when the document was dropped, the store allocated memory couldn't be released. With this PR, branches only hold a weak references, so dropping the document means dropping the store.